rusoto/credential/src/profile.rs

```diff
@@ -34,7 +46,7 @@ impl ProfileProvider {
 
         Ok(ProfileProvider {
```

Comment:
Could this use the new `with_default_configuration` instead?

Reply:
Fixed.